lms/templates/bookmarks/bookmarks_results.underscore

```diff
@@ -0,0 +1,42 @@
+<div id="my-bookmarks" class="sr-is-focusable" tabindex="-1"></div>
+<h2 class="bookmarks-results-header" tabindex="0"><%= gettext("My Bookmarks") %></h2>
```

Comment:
Headings are not focusable elements.  The user woudln't expect that behavior.  If they want to consume all of the content, they use the arrow keys.  If they want to consume only focusable content, they use the tab key.  If they tab and focus lands on the heading, they can only assume that its interactive content, and its not.